### PR TITLE
Responsive header nav and footer

### PR DESCRIPTION
**Pull request by KiaanCastillo**
# Changes
- Cosmetic updates to nav and footer to better match Figma design
- Make nav and footer responsive

### CODE REVIEW

File: pages/resources.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import { Helmet } from "@components";
-import Header from "../components/header.js";
+import { Helmet, HeaderNav } from "@components";
 import Footer from "../components/footer.js";
```
Comment (dannyl1u) [inline pages/resources.js:2]: import { Footer } from "../components/footer.jsx";

to fix compile error

Comment (KiaanCastillo) [reply pages/resources.js]: Thanks for catching that1

**Comment by marcusgchan** [comment]
The navbar isn't centred on larger screens.



Other than that, looks good!

File: styles/components/header-nav.scss [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  color: var(--colour-neutral-000);
+  background: linear-gradient(180deg, #141515 0%, rgba(20, 21, 21, 0) 100%);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  z-index: 10;
+
```
Comment (marcusgchan) [inline styles/components/header-nav.scss:13]: The navbar isn't centred on larger screens.



I think adding this should fix it.
```
left: 0;
right: 0;
 ```


Other than that, looks good!

**Review by marcusgchan — CHANGES_REQUESTED** [review]

File: styles/components/header-nav.scss (hunk 2 above)
Comment (KiaanCastillo) [reply styles/components/header-nav.scss]: Thanks for catching that, Marcus! I see another styling issue for the footer on large screens as well. I should not forget test on large screens from now on 😅

**Review by marcusgchan — APPROVED** [review]
Everything looks good!

**Review by dannyl1u — APPROVED** [review]
LGTM

**Pull request merged**